runtime/libpgmath/lib/common/fltmanip.c

```diff
@@ -79,7 +224,7 @@ __nextafter(double x, double y)
 
   /* x infinity, return -_MAXDOUBLE, +_MAXDOUBLE, or x */
   if ((ixh & 0x7ff00000) == 0x7ff00000) {
-    if ((iyh & 0x7f800000) == 0x7f800000) {
+    if ((iyh & 0x7ff00000) == 0x7ff00000) {
```

Comment:
Normally I would ask you to move it to a separate preliminary commit (with adequate commit message), but for sake of completeness, let's keep it together.

Reply:
I agree with you. We can split this into a separate commit, but keep it in this PR, to reduce the churn.

Reply:
Thanks for your comments, I have splited this into a separate commit.